src/TelegramService/Controllers/BotController.cs

```diff
@@ -0,0 +1,24 @@
+using System.Threading;
+using System.Threading.Tasks;
+using LT.DigitalOffice.TelegramService.Business.Features.Bot;
+using LT.DigitalOffice.TelegramService.Business.Shared.Filters.Bot;
+using Microsoft.AspNetCore.Mvc;
+using Telegram.Bot.Types;
+
+namespace LT.DigitalOffice.TelegramService.Controllers;
+
+[ApiController]
+[Route("[controller]")]
+public class BotController : ControllerBase
+{
+  [HttpPost]
+  [ValidateTelegramBot]
+  public async Task<IActionResult> Post(
+    [FromBody] Update update,
+    [FromServices] UpdateHandler handleUpdateService,
+    CancellationToken cancellationToken)
+  {
+    await handleUpdateService.HandleUpdateAsync(update, cancellationToken);
```

Comment:
I insist on splitting business logic into different handlers so would be nice to create Factory which will accept Update and return handler matching provided message. Even though we will have single endpoint and method in controller, we will have several handlers